packages/fuel-indexer-tests/tests/integration/web_api_sqlite.rs

```diff
@@ -83,3 +88,53 @@ async fn test_database_sqlite_metrics_properly_increments_counts_when_queries_ar
             >= 1,
     );
 }
+
+// #[tokio::test]
```

Comment:
@deekerno Does even this test not work? My issue was with `forc index deploy` but are you saying even this test and `curL` also don't work?

Reply:
Correct. You should be able to get the failure by running `SQLX_OFFLINE=1 cargo run --bin fuel-indexer --release -- --database sqlite --sqlite-database ~/dev/fuel/indexer/packages/fuel-indexer-tests/test.db` and then this `curl` command:
```
 curl -v http://0.0.0.0:29987/api/index/fuel_examples/explorer_index \
    -F "manifest=@examples/block-explorer/explorer_index.manifest.yaml" \
    -F "wasm=packages/fuel-indexer-tests/assets/explorer_index.wasm" \
    -F "schema=@examples/block-explorer/schema/explorer_index.schema.graphql" \
    -H 'Content-type: multipart/form-data' -H "Authorization: foo" | json_pp
```

The asset upload should stall at the `new_graph_root` database operation and an `HTTP 500` should be returned. The same happens in the test.